README.md

```diff
@@ -128,14 +129,16 @@ on the size of the header of 100MB to prevent parsing extremely large JSON.
 - Faster load: PyTorch seems to be the fastest file to load out in the major
 ML formats. However, it does seem to have an extra copy on CPU, which we
 can bypass in this lib [link](https://github.com/huggingface/safetensors/pull/33).
-Currently CPU loading the entire file is still slightly slower than PyTorch on
-some platforms but it's not entirely clear why.
+Currently, CPU loading times are extremely fast with this lib compared to pickle.
+GPU loading times can be sped up but are still hidden behind an environment variable
+(`SAFETENSORS_FAST_GPU=1`) because it hasn't received enough external scrutiny to be safe.
+But it does load roughly 2X faster than PyTorch on regular Linux hardware because of this extra CPU copy skip.
```

Comment:
I took the liberty to modify this as the statement is not True anymore.

Please correct me if the English is wrong.

Reply:
All good from me.